queries/clojure/locals.scm

```diff
@@ -0,0 +1 @@
+; placeholder file to get incremental selection to work
```

Comment:
I guess it would be better if you could just replace this line
https://github.com/sogaiu/nvim-treesitter/blob/908e87216e48385e420367b6e3715d4abb8ee049/lua/nvim-treesitter/configs.lua#L37

by `parsers.has_parser`

Reply:
It's not that simple... But I would still prefer to have this feature for all languages if an empty file is enough.

Reply:
I don't understand how the incremental selection works nor it scope.

Is it easy to explain why locals.scm is necessary for incremental selection?  Or perhaps there is something I might read?

Reply:
The behavior is to use scopes (https://github.com/sogaiu/nvim-treesitter/blob/908e87216e48385e420367b6e3715d4abb8ee049/lua/nvim-treesitter/locals.lua#L261 which reads query files https://github.com/sogaiu/nvim-treesitter/blob/908e87216e48385e420367b6e3715d4abb8ee049/lua/nvim-treesitter/locals.lua#L52) or alternatively the parent.

I guess it could be fixed here: https://github.com/sogaiu/nvim-treesitter/blob/908e87216e48385e420367b6e3715d4abb8ee049/lua/nvim-treesitter/incremental_selection.lua#L86 to only do the call to containing_scope if `query.has_locals(parsers.get_buf_lang()` and else just use `node:parent() or node`.

Reply:
The incremental selection would normally operate on variable scopes e.g. `{ }` of a c++ file

Reply:
Thanks for the explanation.

Reply:
But this isn't in the scope of this PR. Just keep the empty file.